docs/root/version_history/current.rst

```diff
@@ -12,6 +12,11 @@ Minor Behavior Changes
 Bug Fixes
 ---------
 *Changes expected to improve the state of the world and are unlikely to have negative effects*
+* overload: prevent segfault when disabling listener.
+
+  This prevents the stop_listening overload action from causing
+  segmentation faults that can occur if the action is enabled after the
+  listener has already shut down.
```

Comment:
Release notes are typically a single line. See examples at: https://raw.githubusercontent.com/envoyproxy/envoy/master/docs/root/version_history/v1.16.0.rst

For this specific one I suggest the following wording:
listener: fix crash when disabling or re-enabling listeners due to overload while processing LDS updates

Reply:
@antoniovicente Thanks for suggestion! It is corrected now.